python/pyarrow/tests/test_plasma.py

```diff
@@ -292,19 +292,38 @@ def test_put_and_get(self):
             assert result == pa.plasma.ObjectNotAvailable
 
     def test_put_and_get_buffer_without_serialization(self):
+        def do_serialization(meta):
+            return meta != b"RAW"
+        def deserialize_or_output(meta_data):
+            if meta_data[0] == b"RAW":
+                return ray.worker.global_worker.plasma_client.get_bytes_from_buffer(meta_data[1])
+            else:
+                if meta_data[1] == pa.plasma.ObjectNotAvailable:
+                    return pa.plasma.ObjectNotAvailable
+                else:
+                    return ray.worker.global_worker.plasma_client.deserialize_buffer(meta_data[1])
```

Comment:
typo here :) presumably should be `self.plasma_client`? If this test is passing then that suggests that these lines are not hit.